### DISCUSSION TITLE
Create Customize Panel and Dashboard

### DISCUSSION THREAD

**Discussion by amirmmdz** [filamentphp/filament#12159] [Help]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

Hello Fillament Team
I need to customize Filament panel template and change it to my own template.
I want to know how much access I have to change the panel template and can I change the template completely.
Also i want to know can i add my own dashboard page with my own charts to Filament dashboard?

**Comment by leandrocfe** [comment]
Do you mean, the css?
https://filamentphp.com/docs/3.x/panels/themes/#creating-a-custom-theme

You can customize the dashboard page
https://filamentphp.com/docs/3.x/panels/dashboard/#customizing-the-dashboard-page

You can create custom widgets
https://filamentphp.com/docs/3.x/panels/dashboard/#custom-widgets
